scripts/inf-import-state.sh

```diff
@@ -449,7 +449,7 @@ providers="/providers/Microsoft.KeyVault/vaults/$keyVaultId/objectId/$appId"
 module_path="module.enrichmentApp.azurerm_key_vault_access_policy.policy"
 import_resource_if_needed "$module_path" "$resourceId$providers"
 providers="/providers/Microsoft.Web/sites/$appName|$appName"
-module_path="module.enrichmentApp.azurerm_monitor_diagnostic_setting.example"
+module_path="module.enrichmentApp.azurerm_monitor_diagnostic_setting.diagnostic_setting"
```

Comment:
This isn't the correct module that changed.. it is 
![image](https://github.com/microsoft/PubSec-Info-Assistant/assets/48474707/18df878c-073c-44d4-af12-c0c936020bfd)